crowd_anki/representation/note.py

```diff
@@ -117,5 +123,5 @@ def save_to_collection(self, collection, deck, model_map_cache):
 
         if new_note:
             collection.addNote(self.anki_object)
-        else:
+        elif not self.config.import_notes_ignore_deck_movement:
```

Comment:
there is too much of passing things around, I'd prefer if you use a pattern similar to what we've done with `addonManager` here. (making the config one of the `__init__` parameters with None as default value and initializing it via singleton value if nothing is passed in)

Reply:
Yea thought you may say that, no worries. I am inclined to agree :sweat_smile: Done :+1: